wildlifelicensing/apps/dashboard/views/officer.py

```diff
@@ -75,12 +72,25 @@ def _build_tree_nodes(self):
                 self._add_node(assigned_applications_node, node)
         result.append(assigned_applications_node)
 
-        on_behalf_applications = _get_current_onbehalf_applications(self.request.user)
-        if on_behalf_applications.count() > 0:
-            url = reverse_lazy('wl_dashboard:tables_applications_officer_onbehalf')
-            on_behalf_applications_node = self._create_node('My current proxied applications', href=url,
-                                                            count=on_behalf_applications.count())
-            result.append(on_behalf_applications_node)
+        on_behalf_applications_count = \
+            DataTableApplicationsOfficerOnBehalfView._get_proxy_applications(self.request.user).count()
+        # on_behalf_licences_count = DataTableLicencesOfficerOnBehalfView._get_proxy_licences(self.request.user).count()
```

Comment:
Yeah. I first implemented the proxy licence table too. Then decided not to show it, but I left the comment because I have a feeling it might come back. I wonder if I should also delete the views I created for that?